website/pages/confirm.mdx

```diff
@@ -2,6 +2,7 @@
 title: Confirm
 headeronly: true
 searchable: false
+
```

Comment:
Was this intentional?

Reply:
It was to trigger a website build, I'll remove it